jsx/StaticDataTable.js

```diff
@@ -90,8 +94,11 @@ StaticDataTable = React.createClass({
             });
         } else {
             for(var i = 0; i < this.props.Data.length; i += 1) {
-                index.push({ RowIdx: i });
-
+                if (this.props.RowNumCol) {
+                    index.push({ RowIdx: i, Content: this.props.RowNumCol[i]});
```

Comment:
I find this variable name confusing.. can you name it something like RowNameMap, and make sure the type is specified in PropTypes?